[en]update virtnest-vm (#4876)
DaoCloud/DaoCloud-docs: docs/en/docs/virtnest/vm/index.md

This article will explain how to create a virtual machine using two methods: image and YAML file.

Virtual machine, based on KubeVirt, manages virtual machines as cloud native applications,
seamlessly integrating with containers. This allows users to easily deploy virtual machine applications and
enjoy a smooth experience similar to containerized applications.

## Prerequisites

Before creating a virtual machine, make sure you meet the following prerequisites:

- Expose hardware-assisted virtualization to the user operating system.
- [Install virtnest-agent](../install/index.md) on the specified cluster; the operating system kernel version must be 3.15 or higher.
- Create a [namespace](../../kpanda/user-guide/namespaces/createns.md) and [user](../../ghippo/user-guide/access-control/user.md).
- Prepare the image in advance. The platform comes with three built-in images (as shown below). If you need to 

>    create your own image, refer to the open-source project [image creation](https://github.com/Tedezed/kubevirt-images-generator/tree/master).

Reviewer: ```suggestion
  create your own image, refer to [creating from an image with KubeVirt](https://github.com/Tedezed/kubevirt-images-generator/tree/master).
```
